### ISSUE TITLE
Add "pro-tip" column

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma.org#607]
We occasionally have "pro-tips" from editorial practices that will be useful for internet searches, e.g. 

* How to move the content of an ISO Example block to the next line (https://github.com/metanorma/iso-6709/issues/17#issuecomment-1141575621).

I suggest we add a "pro-tip" section that lists them.

**Comment by opoudjis** [comment]
Wherever ZWNJ (&#x200c;) is listed as a way of preventing Asciidoctor applying regexes, that too is a pro-tip.

Can I ask that the editors do the pro-tips instead of me?

**Comment by ronaldtse** [comment]
Of course, @metanorma/editors can you please help gather the previous pro-tips and collect them here? Thanks!

**Comment by manuelfuenmayor** [comment]
Currently, there are two sections in metanorma.org dedicated to list these "pro-tips": [Metanorma AsciiDoc tips](https://www.metanorma.org/author/ref/asciidoc-tips/) and [Edition tips](https://www.metanorma.org/author/editors-guide/edition-tips/).

> Wherever ZWNJ (&#x200c;) is listed as a way of preventing Asciidoctor applying regexes, that too is a pro-tip.

The first section already covers the use of `&#x200c;` to handle particular markup escaping.

@ronaldtse, perhaps the existence of these two sections already covers the requirement of this task. Can I close?
